<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchItem</class>
 <widget class="QWidget" name="SearchItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lab_num">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei Light</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>喜欢的歌</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_ranking">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1245321</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
